myuw/test/api/emaillist.py

```diff
@@ -85,16 +85,11 @@ def test_post_wo_csrf_check(self):
             resp = client.post(
                 url,
                 {u'section_single_A': u'2013,spring,PHYS,122/A',
```

Comment:
Removing them would exclude the case where a primary section instructor requests email lists of any secondary sections taught by TAs.